subprojects/internal-integ-testing/src/test/groovy/org/gradle/integtests/fixtures/logging/StackTraceSanitizingOutputNormalizerTest.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.logging
+
+import org.gradle.exemplar.executor.ExecutionMetadata
+import spock.lang.Specification
+
+class StackTraceSanitizingOutputNormalizerTest extends Specification {
+    def normalizer = new StackTraceSanitizingOutputNormalizer()
+
+    def 'remove harmless stacktraces in android build output'() {
+        given:
+        File dir = File.createTempFile("build-android-app", "")
+        def originalOutput = '''
+Unable to detect AGP versions for included builds. All projects in the build should use the same AGP version. Class name for the included build object: org.gradle.composite.internal.DefaultIncludedBuild$IncludedBuildImpl_Decorated.\t
+Unable to detect AGP versions for included builds. All projects in the build should use the same AGP version. Class name for the included build object: org.gradle.composite.internal.DefaultIncludedBuild$IncludedBuildImpl_Decorated.\t
+File /home/tcagent1/.android/repositories.cfg could not be loaded.\t
+IO exception while downloading manifest:\t
+java.io.FileNotFoundException: https://dl.google.com/android/repository/sys-img/android-desktop/sys-img2-1.xml\t
```

Comment:
Do you know why the plugin tries to download this resource in the first place? Where is this referenced, maybe we can get rid of it completely.

Reply:
No... do you know it? @runningcode

Reply:
I have never heard of this. Is this something new in AGP that 7.X? Could be related to the new instrumentation test infrastructure.

Reply:
This seems to be the license agreement for Android desktop. I found the Google APIs license agreement: https://github.com/eagletmt/android-repository-history/blob/master/repository/sys-img/google_apis/sys-img2-1.xml